swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -1638,18 +1637,18 @@ public void setSSCommon(final SSCommon _ssCommon) {
 	 * call the updateRow.)
 	 *
 	 * @param _mapping         typically a primary key value corresponding the
-	 *                         the displayed option to be updated
-	 * @param _option option that should be updated in the combobox
+                         the displayed currentSelectedOption to be updated
+	 * @param _option currentSelectedOption that should be updated in the combobox
 	 *
 	 * @return returns true if update is successful otherwise returns false.
 	 */
 	public boolean updateOption(final Long _mapping, final String _option) {
 
 		boolean result = false;
 
-		try (Model.Remodel remodel = comboInfo.getRemodel()) {
+		try (Model.Remodel remodel = optionModel.getRemodel()) {
 			final int index = remodel.getMappings().indexOf(_mapping);
-			if (index < 0) {
```

Comment:
Looks like the old code was: 
// 				// PROCEED IF INDEX WAS FOUND
// 				if (index != -1) {

Reply:
So I went to `< 0` (a bug) to `>= 0` (just weird).